doc/releases/release-notes-3.7.rst

```diff
@@ -269,6 +269,10 @@ Drivers and Sensors
 
 * ADC
 
+  * The NXP LPADC driver now supports passing the reference voltage value by using new
+    phandle-array type DT property ``nxp,references``. The DT property ``nxp,reference-supply`` 
+    is now deprecated.
```

Comment:
don't say "deprecated" because that means the old thing is still around, we just changed this. also, keep this in this ADC section unlike the migration guide change